src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -771,19 +642,20 @@ private int getNumberOfHits(String page, String marker, Pattern pattern) throws
      */
     private String getResults(URL source) throws IOException {
 
-        InputStream in = source.openStream();
-        StringBuilder sb = new StringBuilder();
-        byte[] buffer = new byte[256];
-        while (true) {
-            int bytesRead = in.read(buffer);
-            if (bytesRead == -1) {
-                break;
-            }
-            for (int i = 0; i < bytesRead; i++) {
-                sb.append((char) buffer[i]);
+        try(InputStream in = source.openStream()) {
+            StringBuilder sb = new StringBuilder();
```

Comment:
And it would clearly make sense to add an InputStream to String converter. Will look more into it eventually (but not tonight or the next few days). Thanks for the link!